src/registrar/assets/sass/_theme/_admin.scss

```diff
@@ -416,8 +423,40 @@ address.dja-address-contact-list {
         font-size: 0.875rem;
         color: var(--body-quiet-color);
     }
+
+    address button.usa-button__clipboard-link {
+        font-size: 0.875rem !important;
+    }
+
+    td button.usa-button__clipboard-link {
+        font-size: 0.875rem !important;
+    }
```

Comment:
You should be able to consolidate these into one declaration via a comma....while I debated whether we wanted to do this (there is merit to having separate declarations in some situations), it would make sense to consolidate since these are both instances of the same element.

try:
`address button.usa-button__clipboard-link, td button.usa-button__clipboard-link {
        font-size: 0.875rem !important;
    }
`